renovate.json

```diff
@@ -1,5 +1,11 @@
 {
   "extends": [
-    "config:base"
+      "config:base",
+      "group:all",
+      ":disableDependencyDashboard",
+      "schedule:weekly"
```

Comment:
nit: any reason for different indentation? this block seems to have 4 tab indentation while the rest has only 2